### PR TITLE
add shellcheck to the ci and minor cleanup of bash scripts in kube_state_metrics

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

**Comment by mxinden** [comment]
Thanks @tariq1890. Would you mind adding shellcheck to our CI as well? If not, I can do it in a follow up pull request.

See example: https://github.com/coreos/prometheus-operator/blob/master/Makefile#L203

File: .travis.yml [hunk 1]
```diff
@@ -30,8 +33,8 @@ jobs:
     - script: make validate-modules
     # Check that all metrics are documented
     - script: make doccheck
-    # Unit Test
-    - script: make test-unit
+    # Unit Test and shellcheck
+    - script: make shellcheck test-unit
```
Comment (tariq1890) [inline .travis.yml]: added shellcheck to the ci @mxinden :)

**Comment by tariq1890** [comment]
Done! Ready for review

**Comment by tariq1890** [comment]
Phew! Finally got the builds to pass. The issue was basically that of `src/k8s.io/kube-state-metrics` vs `src/github.com/kubernetes/kube-state-metrics` .

File: .travis.yml [hunk 2]
```diff
@@ -21,6 +24,7 @@ env:
 install:
 - mkdir -p $HOME/gopath/src/k8s.io
 - mv $TRAVIS_BUILD_DIR $HOME/gopath/src/k8s.io/kube-state-metrics
+- cd $HOME/gopath/src/k8s.io/kube-state-metrics
```
Comment (tariq1890) [inline .travis.yml]: This step was needed to get the builds to pass

**Comment by tariq1890** [comment]
No worries at all! Happy to hell :)

**Pull request merged**